src/Types.hs

```diff
@@ -258,6 +261,9 @@ instance ToSchema PoolOfflineMetadata
 newtype PoolMetadataWrapped = PoolMetadataWrapped Text
     deriving (Eq, Ord, Show, Generic)
 
+instance MimeUnrender OctetStream PoolMetadataWrapped where
```

Comment:
Fine.

Reply:
Yes... I chose this option, because the metadata is hashed as bytes, not at json content. So a newline or a whitespace will change the hash.